Swift/KVSiOSApp/WebRTCClient.swift

```diff
@@ -83,6 +86,8 @@ final class WebRTCClient: NSObject {
             remoteVideoTrack = nil
             peerConnection.remove(stream)
         }
+        peerConnectionFoundMap.removeAll();
```

Comment:
If this is more like setting up, should this be moved to the init itself?

Reply:
This is more like cleanup at close. Emptying the maps.

Reply:
Given its a configure method, its not good to have cleaning up here. Can we move this to a different method in the lifecycle?

Reply:
Sure. What do you think is a better place to clean this map up? 
I am looking to add it in a place where we terminate the session.